java/main/com/google/privacy/differentialprivacy/GaussianNoise.java

```diff
@@ -114,6 +114,32 @@ public MechanismType getMechanismType() {
     return MechanismType.GAUSSIAN;
   }
 
+  /**
+   * Computes a confidence interval that contains the raw value {@code x} passed to
+   * {@link #addNoise(double, int, double, double, Double)} with a probability greater
+   * or equal to {@code confidenceLevel} based on the specified {@noisedValue} and noise parameters.
```

Comment:
sorry, this was my mistake earlier, but this should be "{@link noisedValue}"

For consistency with the addNoise method, I would suggest calling the parameter "noisedX" rather than "noisedValue". I think this make the connection between the two easier to see.

Could you please also fix this in the other docs

Reply:
I added it {@code noisedX}, should be fine now.